unit_tests/test_config.py

```diff
@@ -299,6 +300,7 @@ def test_15_new_scylla_repo(self):
         os.environ['SCT_SCYLLA_REPO'] = centos_repo
         os.environ['SCT_NEW_SCYLLA_REPO'] = centos_repo
         os.environ['SCT_USER_PREFIX'] = 'testing'
+        os.environ['SCT_N_DB_NODES'] = '3'
```

Comment:
@fruch 
I got an error because this variable contains one value...

Reply:
why did you added it ?

Reply:
The test failed because the `TypeError`:
```
TypeError: int() argument must be a string, a bytes-like object or a number, not 'NoneType'
```